operator/src/main/java/com/datastax/oss/kaap/controllers/function/FunctionsWorkerResourcesFactory.java

```diff
@@ -299,7 +299,7 @@ public void patchConfigMap() {
                         "pulsarRootDir", "/pulsar",
                         "submittingInsidePod", true,
                         "pulsarServiceUrl", brokerServiceUrl,
-                        "pulsarAdminUrl", "https://%s.%s:6750/"
+                        "pulsarAdminUrl", "https://%s.%s:6751/"
```

Comment:
Given that line 301 has `brokerServiceUrl`, I think we might need to dynamically compute this value.

Reply:
It seems to me that are a lot of little things to do in this class like making use of constants for the ports provided as ints at the top, but likely better as strings.

Reply:
Yes it has to be computed based on tls configuration